src/examples/libpmemobj/pmemlog/obj_pmemlog.c

```diff
@@ -328,13 +329,14 @@ pmemlog_walk(PMEMlogpool *plp, size_t chunksize,
 static int
 process_chunk(const void *buf, size_t len, void *arg)
 {
-	char tmp[len + 1];
+	char *tmp = malloc(len + 1);
```

Comment:
1. check for error
2. instead of allocaing and freeing this buffer in a loop, I'd recommend its allocation outside of pmemlog_walk routine and passing it in as an argument.